packages/vite/jest.config.ts

```diff
@@ -3,7 +3,7 @@ const fs = require('fs');
 // Reading the SWC compilation config and remove the "exclude"
 // for the test files to be compiled by SWC
 const { exclude: _, ...swcJestConfig } = JSON.parse(
-  fs.readFileSync(`${__dirname}/.lib.swcrc`, 'utf-8')
+  fs.readFileSync(`${__dirname}/.swcrc`, 'utf-8')
 );
 
```

Comment:
@nartc was this supposed to set for swcrc to false?
https://github.com/nrwl/nx/blob/ad26b55672c6877cf87739be0f1618bda96d772f/packages/js/src/migrations/update-15-8-0/rename-swcrc-config.ts#L100-L102
```ts
// disable .swcrc look-up by SWC core because we're passing in swcJestConfig ourselves.
// If we do not disable this, SWC Core will read .swcrc and won't transform our test files due to "exclude"
if (swcJestConfig.swcrc === undefined) {
  swcJestConfig.swcrc = false;
}
```